cmd/openshift/operator/kodata/static/tekton-results/logs-rbac/rbac.yaml

```diff
@@ -58,7 +75,7 @@ metadata:
 roleRef:
   apiGroup: rbac.authorization.k8s.io
   kind: ClusterRole
-  name: cluster-logging-application-view
```

Comment:
Don't we need to delete the role with this name ?

Reply:
Nope. We aren't creating the role. It was something that we were leveraging but under certain conditions, it isn't being created by Logging operator.